UM/PackageManager.py

```diff
@@ -296,6 +296,17 @@ def getAllInstalledPackageIDs(self) -> Set[str]:
 
         return all_installed_ids
 
+    def getUserInstalledPackagesOnMarketplace(self) -> List[Tuple[str, str]]:
```

Comment:
Shouldn't it just be "getUserInstalledPackages" or "getUserInstalledPackagesAndVersions"? (or somethinglike that)

Uranium has no notion of a Marketplace existing after all.

Reply:
I'm very much open to name/variable suggestion. This is the first that came onto my mind, considering the other names in that file.
I'm also considering about refactoring few functions from there so that we'd get better functions names and less duplicated code, but I guess that's a topic for another day.

Reply:
To stay consistent with the variable names that I've used in Cura, I'm changing it to "getUserSubscribedPackagesAndVersions".